chargebee/src/main/java/com/chargebee/android/restore/CBRestorePurchaseManager.kt

```diff
@@ -65,7 +66,10 @@ class CBRestorePurchaseManager {
                     retrieveRestoreSubscription(purchaseToken, {
                         restorePurchases.add(it)
                         when (it.storeStatus) {
-                            StoreStatus.Active.value -> activeTransactions.add(storeTransaction)
+                            StoreStatus.Active.value -> {
+                                activeTransactions.add(storeTransaction)
+                                allTransactions.add(storeTransaction)
```

Comment:
This can get difficult to follow. Can we remove the static/companion variables and pass the list as parameters instead?